waterlog/src/Stylesheets/_progressBar.scss

```diff
@@ -5,15 +5,44 @@
   margin: 0 10px;
 }
 
-.filler {
-  background-color: red;
+.filler-low {
+  background-color: #FFEA00 !important;
```

Comment:
don't use important unless you are overriding inline styles used by component libraries.

Reply:
Done please re-check